### PR TITLE
create mp-b2b tests for merchant-opening-hours

### PR DESCRIPTION
**Pull request by m-alama**
### PR Description

create mp-b2b tests for merchant-opening-hours

### CODE REVIEW

File: resources/environments/environments_api_mp_b2b.json [hunk 1]
```diff
@@ -0,0 +1,396 @@
+{
+    "environment": "api_b2b",
```
Comment (vadym-uskov) [inline resources/environments/environments_api_mp_b2b.json:2]: change env value to 'api_mp_b2b'

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/negative.robot [hunk 2]
```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+
+*** Test Cases ***
```
Comment (vadym-uskov) [inline tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/negative.robot:8]: please add test without merchant Id to check status code #400

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot [hunk 3]
```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
```
Comment (vadym-uskov) [inline tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:7]: Please add test with include: /merchants/${merchant_id}?include=merchant-opening-hours

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ENABLER
+    TestSetup
+
+Retrieves_merchant_opening_hours
+    When I send a GET request:  /merchants/${merchant_id}/merchant-opening-hours
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Each array element of array in response should contain property:    [data]    type
+    And Each array element of array in response should contain property:    [data]    id
```
Comment (vadym-uskov) [inline tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:16]: - we can check type and id values
- also there is no need to use 'Each array element of array in response should contain property:' if we have only 1 element in [data] array

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    And Each array element of array in response should contain nested property with datatype:    [data]    [attributes][dateSchedule]    list
+    And Each array element of array in response should contain nested property:    [data]    [attributes][weekdaySchedule]    day
+    And Each array element of array in response should contain nested property:    [data]    [attributes][weekdaySchedule]    timeFrom
+    And Each array element of array in response should contain nested property:    [data]    [attributes][weekdaySchedule]    timeTo
+    And Each array element of array in response should contain nested property:    [data]    [attributes][dateSchedule]    date
+    And Each array element of array in response should contain nested property:    [data]    [attributes][dateSchedule]    timeFrom
+    And Each array element of array in response should contain nested property:    [data]    [attributes][dateSchedule]    timeTo
+    And Each array element of array in response should contain nested property:    [data]    [attributes][dateSchedule]    noteGlossaryKey
+    And Each array element of array in response should contain nested property:    [data]    [links]    self
```
Comment (vadym-uskov) [inline tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:28]: it's better to use 'Response body has correct self link' keyword for links

Comment (m-alama) [reply tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:28]: Done

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/negative.robot (hunk 2 above)
Comment (m-alama) [reply tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/negative.robot:8]: Done

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot (hunk 3 above)
Comment (m-alama) [reply tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:7]: Done

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot (hunk 4 above)
Comment (m-alama) [reply tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot:16]: Done

File: tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot (hunk 5 above)
Comment (vadym-uskov) [reply tests/api/mp_b2b/glue/merchants_endpoints/merchant_opening_hours/positive.robot]: There is no need in line 28:
And Each array element of array in response should contain nested property:    [data]    [links]    self

It's same to And Response body has correct self link

File: tests/api/mp_b2b/glue/merchant_endpoints/merchant_opening_hours/positive.robot [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    And Response Should Contain The Array Larger Than a Certain Size:    [data][relationships]    0
+    And Response body parameter should not be EMPTY:    [data][relationships]
+    And Response body parameter should not be EMPTY:    [data][relationships][merchant-opening-hours]
+    And Each Array Element Of Array In Response Should Contain Property:    [data][relationships][merchant-opening-hours][data]    type
+    And Each Array Element Of Array In Response Should Contain Property:    [data][relationships][merchant-opening-hours][data]    id
+    And Response body parameter should not be EMPTY:    [included]
+    And Each Array Element Of Array In Response Should Contain Property:    [included]    type
+    And Each Array Element Of Array In Response Should Contain Property:    [included]    id
+    And Each Array Element Of Array In Response Should Contain Property:    [included]    attributes
+    And Each Array Element Of Array In Response Should Contain Property:    [included]    links
```
Comment (vadym-uskov) [inline tests/api/mp_b2b/glue/merchant_endpoints/merchant_opening_hours/positive.robot:49]: You can also check concrete details for included parameters, please update branch from master and than you can use existing variables for the current merchant

**Review by vadym-uskov — DISMISSED** [review]

**Review by vadym-uskov — APPROVED** [review]

**Pull request merged**